### ISSUE TITLE
Update `single-line-log` dependency

### ISSUE THREAD

**Issue by kgryte** [maxogden/nugget#18]
The `single-line-log` dependency as required by this module is [__unlicensed__](https://github.com/freeall/single-line-log/tree/38a2ea84366d22810d915c0fbd9cf9d0626927c7). More recent versions of `single-line-log` are [licensed](https://github.com/freeall/single-line-log/tree/515b3b99b699396c2ad5f937e4b490b6f9fbff0e).

Background: this was discovered upon doing a license audit. The trail was `electron` -> `electron-download` -> `nugget` -> `single-line-log` (old version). The use of unlicensed code creates legal issues, not just for this module, but also for anyone who depends on this module.

Note: the `README` for `single-line-log@0.4.1` states that the license is MIT; however, this file was not published to npm. As a result, the code as distributed lacks license information.

```
{
  "author": {
    "name": "Tobias Baunbæk",
    "email": "freeall@gmail.com"
  },
  "bugs": {
    "url": "https://github.com/freeall/single-line-log/issues"
  },
  "dependencies": {},
  "description": "Keep writing to the same line in the terminal. Very useful when you write progress bars, or a status message during longer operations",
  "devDependencies": {},
  "directories": {},
  "dist": {
    "shasum": "87a55649f749d783ec0dcd804e8140d9873c7cee",
    "tarball": "https://registry.npmjs.org/single-line-log/-/single-line-log-0.4.1.tgz"
  },
  "homepage": "https://github.com/freeall/single-line-log",
  "keywords": [
    "single",
    "line",
    "log",
    "output",
    "overwrite",
    "collapse",
    "stdout",
    "terminal",
    "tty",
    "cli",
    "shell"
  ],
  "maintainers": [
    {
      "name": "freeall",
      "email": "freeall@gmail.com"
    }
  ],
  "name": "single-line-log",
  "optionalDependencies": {},
  "readme": "ERROR: No README data found!",
  "repository": {
    "type": "git",
    "url": "git://github.com/freeall/single-line-log.git"
  },
  "version": "0.4.1"
}
```

**Comment by mafintosh** [comment]
fixed in 2.0.1

**Comment by kgryte** [comment]
Thanks @mafintosh !
